designdocs/schema_v2.md

```diff
@@ -0,0 +1,41 @@
+# On Chain Message Storage
```

Comment:
Might be nice to update this doc with some numbers, ie before/after PoV weights. Maybe add a placeholder section for now?